### PR TITLE
FEAT: process `make/to/mold` handle! more better

### PR DESCRIPTION
**Pull request by bitbegin**

### CODE REVIEW

File: runtime/datatypes/handle.reds [hunk 1]
```diff
@@ -53,6 +53,36 @@ handle: context [
...
+
+	to: func [
+		proto 	[red-value!]							;-- overwrite this slot with result
+		spec	[red-value!]
+		type	[integer!]
+		return: [red-value!]
+		/local
+			h	[red-value!]
+	][
+		#if debug? = yes [if verbose > 0 [print-line "to/make"]]
```
Comment (greggirwin) [inline runtime/datatypes/handle.reds]: Should this line print `handle/to` rather than `to/make`?

**Comment by qtxie** [comment]
The handle! value is hidden on purpose. The limitation of `make` and `to` are on purpose too.

**Comment by dockimbel** [comment]
`handle!` type exists to make Red able to pass around an opaque pointer for low-level needs, without letting the user modify it at Red level. Those changes are breaking the purpose of `handle!` type, so they will be rejected.

**Comment by bitbegin** [comment]
this PR just for test, as it failed the ci, related to #3956

**Pull request closed without merging**